packages/styles/scss/patterns/sub-patterns/card/index.scss

```diff
@@ -78,6 +99,28 @@
       padding: 0;
     }
   }
+
+  .#{$prefix}--card--inverse {
+    background-color: $inverse-02;
+
+    .#{$prefix}--card__title,
+    .#{$prefix}--card__content {
+      color: $inverse-01;
+    }
+
+    .#{$prefix}--card__eyebrow {
+      color: $text-03;
+    }
+
+    &.#{$prefix}--card--link:hover {
+      background-color: $inverse-hover-ui;
+    }
+
+    .#{$prefix}--card__cta,
+    .#{$prefix}--card__cta a:visited {
+      color: $inverse-link;
+    }
+  }
```

Comment:
We need to set time aside to touch base to make sure the updates y'all made are reflected within the functional specs.